packages/mdc-segmented-button/segment/constants.ts

```diff
@@ -29,7 +29,9 @@ export const strings = {
   ARIA_PRESSED: 'aria-pressed',
   DATA_SEGMENT_ID: 'data-segment-id',
   TRUE: 'true',
-  FALSE: 'false'
+  FALSE: 'false',
```

Comment:
Instead of grouping constants by their type I would group by feature.

For examples, attributes, events, ariaValues, etc.